### DISCUSSION TITLE
How to restart animation?

### DISCUSSION THREAD

**Discussion by brimby** [Abhinandan-Kushwaha/react-native-gifted-charts#733] [Q&A] [answered]
I've added isAnimated to my lineChart, and I see it animate when the chart first loads, so that's good. BUT when I simply change the data array by way of setting the state, the chart changes abruptly without animation. Is there something I can do to trigger the animation when the data prop changes?

**Comment by Abhinandan-Kushwaha** [comment, marked as answer]
Hi @brimby 
Did you try the `animateOnDataChange` prop?
